CHANGELOG.md

```diff
@@ -105,7 +105,7 @@ Fixes #63
 
 ### Other Changes
 
-- changelog_to_tag action - support other than "master" for the main branch name (#66)
+- changelog_to_tag action - support other than "master" for the main branch name (#66) // wokeignore:rule=master
```

Comment:
this change is unrelated?

Reply:
Oops, sorry. I removed the diff from the commit.